src/core/actions.ts

```diff
@@ -124,6 +124,12 @@ namespace Actions {
   export type ReceiveRedirect = Action<typeof RECEIVE_REDIRECT, string>;
   export const RECEIVE_RECOMMENDATIONS_PRODUCTS = 'RECEIVE_RECOMMENDATIONS_PRODUCTS';
   export type ReceiveRecommendationsProducts = Action<typeof RECEIVE_RECOMMENDATIONS_PRODUCTS, Store.Product[]>;
+  export const RECEIVE_RECOMMENDATIONS_REFINEMENTS = 'RECEIVE_RECOMMENDATIONS_REFINEMENTS';
+  // tslint:disable-next-line max-line-length
+  export type ReceiveRecommendationsRefinements = Action<typeof RECEIVE_RECOMMENDATIONS_REFINEMENTS, Store.Recommendations.Navigation[]>;
+  export const RECEIVE_RECOMMENDATIONS_NAVIGATIONS = 'RECEIVE_RECOMMENDATIONS_NAVIGATIONS';
```

Comment:
I'd prefer to name this (and the associated action creator, saga, etc.) to `ReceiveNavigationSort` (as this is the data we're actually working with)